src/components/Dashboard/StaffNotation.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="staff-notation" class="pr-6" :class="{ 'add-scrollbar-offset': addScrollbarOffset }">
```

Comment:
Small fix per Yui: don't offset this, so that it's in line with the buttons in the Filing History List.